lib/vectortilefeature.js

```diff
@@ -125,3 +125,36 @@ VectorTileFeature.prototype.bbox = function() {
 
     return [x1, y1, x2, y2];
 };
+
+var SphericalMercator = require('sphericalmercator');
+
+VectorTileFeature.prototype.toGeoJSON = function(x, y, z) {
+    var merc = new SphericalMercator({size: this.extent}),
+        x0 = this.extent * x,
+        y0 = this.extent * y,
+        coords = this.loadGeometry(),
+        type = VectorTileFeature.types[this.type];
+
+    for (var i = 0; i < coords.length; i++) {
+        var line = coords[i];
+        for (var j = 0; j < line.length; j++) {
+            var p = line[j];
+            line[j] = merc.ll([p.x + x0, p.y + y0], z);
```

Comment:
I knew I could count on you to ask this question! :) Inlined the math in 1d3f4c1ebb00565919a8c8f05f93e9340b22a297.